### PR TITLE
Document SHIELD study datasets

### PR DESCRIPTION
**Pull request by tdsmith**
Here's a first pass at identifying datasets that are useful for analysts working on SHIELD studies.

This looks mostly complementary to #164 but some cross-references could be useful once either lands.

I'm conflicted about whether it would be useful and in-scope to provide more context or examples for how to e.g. look at enrollment or unenrollment events using the `events` table; what do y'all think?

### CODE REVIEW

File: datasets/shield.md [hunk 1]
```diff
@@ -1,5 +1,121 @@
...
+
+## Table of contents
+
+<!-- toc -->
+
+## Dashboards
+
+[Firefox Test Tube](https://firefox-test-tube.herokuapp.com/)
+shows live enrollment activity and a selection of browser activity metrics
+for active pref-flip (but not add-on) experiments.
```
Comment (sunahsuh) [inline datasets/shield.md]: That's.. not strictly true :/ We show add-on experiments if their normandy name matches with the experiment name that the addon registers as, which doesn't happen all the time.

File: datasets/shield.md [hunk 2]
```diff
@@ -1,5 +1,121 @@
...
+### `events`
+
+The [`events` table](../datasets/batch_view/events/reference.html) includes
+Normandy enrollment and unenrollment events.
+
+Normandy events have event category `normandy`.
+The event value will contain the experiment slug.
+
+The event schema is described
+[in the Firefox source tree](https://hg.mozilla.org/mozilla-central/file/c4c125ee2556/toolkit/components/normandy/lib/TelemetryEvents.jsm#l12).
```
Comment (sunahsuh) [inline datasets/shield.md]: Re: linking to a specific revision
Pro: the line number doesn't change
Con: the file could change and we'd have no idea that the link was out of date

Not sure what's better here, but I'm kind of inclined to link to latest and have people flag when it's pointing to the wrong thing?

File: datasets/shield.md [hunk 3]
```diff
@@ -1,5 +1,121 @@
...
+[`mozilla-pipeline-schemas` repository](https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/dev/schemas/telemetry/shield-study-addon).
+
+The key-value pairs are present in `data` attribute of the `payload` column.
+
+The `telemetry_shield_study_addon_parquet` table is produced by direct-to-parquet;
+data latency should be less than 1 hour.
+
+### `telemetry_shield_study_parquet`
+
+Telemetry from Normandy itself, containing enrollment and unenrollment events.
```
Comment (sunahsuh) [inline datasets/shield.md]: Is it actually Normandy emitting these and not shield-study-utils?

File: datasets/shield.md [hunk 4]
```diff
@@ -1,5 +1,121 @@
...
+These events also appear in the `events` table.
+
+The `telemetry_shield_study_parquet` table is produced by direct-to-parquet;
+data latency should be less than 1 hour.
+
+## Ping datasets
```
Comment (sunahsuh) [inline datasets/shield.md]: Technically telemetry-cohorts a "source". I'd consider using the "Raw ping" terminology, which is commonly used for when we have to drop down to this level

File: datasets/shield.md [hunk 5]
```diff
@@ -1,5 +1,121 @@
...
+
+## Dashboards
+
+[Firefox Test Tube](https://firefox-test-tube.herokuapp.com/)
+shows live enrollment activity and a selection of browser activity metrics
+for active pref-flip (but not add-on) experiments.
+
+## Experiment slugs
+
+Each experiment is associated with a slug,
```
Comment (sunahsuh) [inline datasets/shield.md]: One normandy quirk is that every pref-flip experiment has a slug, while addon studies have a "name". Not sure why this is the case but so it goes.

File: datasets/shield.md [hunk 6]
```diff
@@ -1,5 +1,121 @@
...
+which is the label used to identify the experiment to Normandy clients.
+The slug is also used to identify the experiment in telemetry.
+
+You can determine the slug for a particular experiment by consulting
+[Test Tube](https://firefox-test-tube.herokuapp.com/) (only for pref-flip experiments),
+[this summary table](https://metrics.mozilla.com/~sguha/report/normandy_recipes.html),
```
Comment (sunahsuh) [inline datasets/shield.md]: There's also https://strategy-and-insights.mozilla.com/shield-studies/index.html

File: datasets/shield.md (hunk 5 above)
Comment (tdsmith) [reply datasets/shield.md]: Can I assert that add-ons have a slug specified by a `name` field?

File: datasets/shield.md (hunk 6 above)
Comment (tdsmith) [reply datasets/shield.md]: Added this and Experimenter to the dashboards section -- neither of these actually displays slugs!

File: datasets/shield.md (hunk 4 above)
Comment (tdsmith) [reply datasets/shield.md]: Thanks; I wasn't sure what to call them -- took another crack at this

File: datasets/shield.md (hunk 2 above)
Comment (tdsmith) [reply datasets/shield.md]: Linking the latest sgtm

File: datasets/shield.md (hunk 1 above)
Comment (tdsmith) [reply datasets/shield.md]: Added a caveat

File: datasets/shield.md (hunk 3 above)
Comment (tdsmith) [reply datasets/shield.md]: Whoops, you're right. Will Normandy still emit events that get collected to the `events` table for these?

Comment (tdsmith) [reply datasets/shield.md]: Yes, it will.

**Comment by tdsmith** [comment]
Great, thanks! I'll land this now and think about the common experiment queries cookbook.

**Pull request merged**